tests/browser/browser.js

```diff
@@ -168,3 +168,45 @@ async function getLocationObject (page) {
   })
   return JSON.parse(location)
 }
+
+describe('platform specific content', () => {
```

Comment:
I think these tests would be faster under content or rendering tests, using Cheerio instead of puppeteer.

Reply:
The tests aren't that slow compared to other ones, but I can take a look if I can port this over.

Reply:
Upon review, I don't think that there is a way to test the user agent detection / overriding code with Cheerio. The only test that can be ported IMO is `should have a platform switcher`, which is purely rendering-based.

Reply:
Actually, that test also relies on JavaScript execution in the third expect(). The class `selected` for the active tab isn't rendered but added at runtime: https://github.com/github/docs/blob/0c8f74fa4307be81de0d1d1e4ecf88148c3e3553/javascripts/display-platform-specific-content.js#L35

Reply:
I also can't seem to get jest to execute my rendering test. Does it require the test server to run at port 4001 or something? I keep getting promise resolve timeouts but don't do anything different to existing tests.